demo/src/app/home/home.page.html

```diff
@@ -10,13 +10,57 @@
 </ion-header>
 
 <ion-content [fullscreen]="true">
+  <div class="content-heading">
+    <ion-button [disabled]="remainsMediaToUpload.length < 1" color="primary" (click)="uploadAll()">Start all</ion-button>
+  </div>
   <ion-list>
     <ion-item *ngFor="let media of allMedia">
-      <img [src]="media.uri" style="width: 50px; height:auto;" />
-      <div class="item-text-center item-text-wrap">
-          {{media.status}}
+      <div class="media-card">
+        <div class="thumbnail">
+          <img [src]="media.local_uri" />
```

Comment:
```suggestion
          <img [src]="media.localUri" />
```
et les changements qui vont avec